### DISCUSSION TITLE
Python and Golang alternatives

### DISCUSSION THREAD

**Discussion by draew6** [openapi-ts/openapi-typescript#1491] [General]
Do you know about something like your fetch in python and golang ecosystems or this is very unique? Not sure if i am searching right way but did not find anything but maybe i dont know how it's called.

**Comment by drwpow** [comment]
That’s a great question. Typically with Python and Go you’re dealing with backend code anyway so size isn’t an issue. You’d probably use what’s called “OpenAPI codegen” where hundreds of functions are generated for your schema. For Python, though, the typing isn’t as strict because, well, it’s Python. But Go has many great OpenAPI utils.

But the zero client weight, static type inference is possible in TypeScript thanks to static typechecking and generics. Neither Python nor Go have both of those. But you could probably recreate this with any other typed language that has both (e.g. Rust)

**Reply by draew6** [reply]
Thanks for reply. I see.
openapi generators I saw are either not working for my API, confusing or just bad experience :/ (python and JS)
golang from quick search looks better as there is more tools but did not test it yet and looks like i would have to do more work. I wanted to go out from TS but I think your library is only reason I will keep TS in stack (for BFF).  It's so nice, effective and simple. I will try to use it with deno to avoid node. 
Thank you for this package. I remember when old repo had 20 stars and couldnt believe I found something so amazing.
